packages/suite/src/hooks/wallet/trading/form/common/useTradingFormActions.ts

```diff
@@ -72,7 +72,7 @@ export const useTradingFormActions = <T extends TradingSellExchangeFormProps>({
         deviceState: device?.state?.staticSessionId,
     });
     const isNotFormPage = pageType !== 'form';
-    const [fractionButton, setFractionButton] = useState<number | undefined>(undefined);
+    const [fractionButton, setFractionButtonState] = useState<number | undefined>(undefined);
```

Comment:
```suggestion
    const [fractionButton, setFractionButton] = useState<number | undefined>(undefined);
```
is there reason to rename just `setter`?